instrumentation/elastic-opentelemetry-instrumentation-openai/tests/test_chat_completions.py

```diff
@@ -1166,4 +1166,8 @@ async def test_async_local_connection_error(self):
         self.assertEqual(span.events, ())
 
         (operation_duration_metric,) = self.get_sorted_metrics()
-        self.assertErrorOperationDurationMetric(operation_duration_metric, {"error.type": "APIConnectionError"})
+        self.assertErrorOperationDurationMetric(
+            operation_duration_metric,
+            {"error.type": "APIConnectionError"},
+            data_point=0.0072969673201441765,
```

Comment:
non-blocking Q: Where does that number come from?  AFAICT that data_point is used to create a histogram metric and the value(s) compared with a delta of 0.5. So is this value saying anything other than "the expected `operation_duration_metric` is approximately between -0.5 and +0.5?

Reply:
Yes, the magic number comes from local runs, the delta comes from faillures in CI :)